src/js/merkle-drop/ClaimFlow.js

```diff
@@ -32,10 +38,11 @@ const STATE = {
 
 function ClaimFlow() {
   const [internalState, setInternalState] = useState(STATE.INPUT)
-  const [address, setAddress] = useState("")
+  const [merkleDropAddress, setMerkleDropAddress] = useState("")
```

Comment:
According to other parts in the project this name refers to the merkle drop contract address. Maybe we could use `eligibleAddress`? But actually this already includes a statement that must not be correct. :thinking:

Reply:
yeah, the idea was to differentiate it from the web3 address

Reply:
I will change it back to address